src/rpdk/core/contract/resource_client.py

```diff
@@ -837,6 +837,12 @@ def metadata_contains_tag_property(self):
         except KeyError:
             return False
 
+    def metadata_contains_tag_permissions(self):
+        try:
+            return "permissions" in self._schema["tagging"]
+        except KeyError:
+            return False
+
```

Comment:
do we need it?
```suggestion
    def metadata_contains_tag_permissions(self):
        return "permission" in self._schema.get("tagging", {})
```

Reply:
lets add a method that can access actual permission from the list + add appropriate unit test for that. Other than that it looks good to me